samples/retry-policy/RetryPolicyExample.cs

```diff
@@ -0,0 +1,25 @@
+namespace MyNamespace;
+
+using Temporalio.Workflows;
+
+[Workflow]
+public class RetryPolicyExample
```

Comment:
Where is this class used? Is there anything to confirm it works/compiles? It may not even belong to a project.